# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/mol/kernel-name-clash.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mol-2003-06-20/src/shared/asmdefs.h.orig	2003-06-12 13:00:59.000000000 +0200
+++ mol-2003-06-20/src/shared/asmdefs.h	2003-09-16 13:02:46.000000000 +0200
@@ -83,26 +83,26 @@
 	addi	_dreg,_dreg,LO(_addr)
 ])
 
-MACRO(LOAD_GPR_RANGE, [start, end, offs, base], [
-	mFORLOOP([i],0,31,[ .if (i >= _start) & (i <= _end)
+MACRO(LOAD_GPR_RANGE, [istart, iend, offs, base], [
+	mFORLOOP([i],0,31,[ .if (i >= _istart) & (i <= _iend)
 		lwz rPREFIX[]i,_offs+i[]*4(_base)
 	.endif 
 ])])
 
-MACRO(STORE_GPR_RANGE, [start, end, offs, base], [
-	 mFORLOOP([i],0,31,[ .if (i >= _start) & (i <= _end)
+MACRO(STORE_GPR_RANGE, [istart, iend, offs, base], [
+	 mFORLOOP([i],0,31,[ .if (i >= _istart) & (i <= _iend)
 		stw rPREFIX[]i,_offs+i[]*4(_base)
 	.endif 
 ])])
 
-MACRO(LOAD_FPR_RANGE, [start, end, offs, base], [
-	mFORLOOP([i],0,31,[ .if (i >= _start) & (i <= _end)
+MACRO(LOAD_FPR_RANGE, [istart, iend, offs, base], [
+	mFORLOOP([i],0,31,[ .if (i >= _istart) & (i <= _iend)
 		lfd fPREFIX[]i,_offs+i[]*8(_base)
 	.endif
 ])])
 
-MACRO(STORE_FPR_RANGE, [start, end, offs, base], [
-	mFORLOOP([i],0,31,[ .if (i >= _start) & (i <= _end)
+MACRO(STORE_FPR_RANGE, [istart, iend, offs, base], [
+	mFORLOOP([i],0,31,[ .if (i >= _istart) & (i <= _iend)
 		stfd fPREFIX[]i,_offs+i[]*8(_base)
 	.endif 
 ])])
